ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -240,138 +240,131 @@
   </Form::Field::ListWrapper>
 
   {{! Account Claim Maps }}
-  <Rose::Form::Fieldset>
-    <:title>{{t 'form.account_claim_maps.label'}}</:title>
-    <:description>{{t 'form.account_claim_maps.help'}}</:description>
-    <:body>
-      {{#if @model.errors.account_claim_maps}}
-        <Rose::Form::Errors as |errors|>
-          {{#each @model.errors.account_claim_maps as |error|}}
-            <errors.message>{{error.message}}</errors.message>
-          {{/each}}
-        </Rose::Form::Errors>
-      {{/if}}
-      <Rose::Table @style='condensed' as |table|>
-        <table.header as |header|>
-          <header.row as |row|>
-            <row.headerCell>{{t 'form.from_claim.label'}}</row.headerCell>
-            <row.headerCell>{{t 'form.to_claim.label'}}</row.headerCell>
-            <row.headerCell>{{t 'titles.actions'}}</row.headerCell>
-          </header.row>
-        </table.header>
-        <table.body as |body|>
-          {{#each @model.account_claim_maps as |claimMap i|}}
-            <body.row as |row|>
-              <row.cell>
-                <form.input
+  <Hds::Form::Fieldset @isOptional={{true}} as |F|>
+    <F.Legend>{{t 'form.account_claim_maps.label'}}</F.Legend>
+    <F.HelperText>{{t 'form.account_claim_maps.help'}}</F.HelperText>
+    {{#let (unique-id) (unique-id) as |fromHeaderID toHeaderID|}}
+      <F.Control>
+        <Hds::Table class='list-wrapper-field'>
```

Comment:
I am getting a bit confused by the answers here. I will try to clarify:

- [Why are we adding ID's to the TH](https://github.com/hashicorp/boundary-ui/pull/2136/files#r1489761821)? So we can identify them, for aria purposes, the TH cell titles are labels to the TR cells inputs. Therefore, they need identification so we can reference them properly.

- [Usage of `unique-id`](https://github.com/hashicorp/boundary-ui/pull/2136/files#r1492633347): [Documentation about unique-id here](https://api.emberjs.com/ember/4.12/classes/Ember.Templates.helpers/methods/each?anchor=unique-id). NONE BLOCKING COMMENT: Do we need unique-id in this use case? it is fine to leave it, but we could also had use a human readable identifier, since the purpose of adding the identifier is aria. I.E: `id=fromClaim`. 

Are these assumptions correct?

Reply:
@calcaide that is correct and I totally agree, I have a commit coming to fix this.